d2game/d2player/mini_panel.go

```diff
@@ -262,7 +262,7 @@ func (m *miniPanel) Toggle() {
 }
 
 func (m *miniPanel) Open() {
-	if !m.movedLeft && !m.movedRight {
+	if !(m.movedRight && m.movedLeft) {
```

Comment:
Looks good! Thanks :)

Reply:
one bracket, but a big change :smile: Thanx for review @juander-ux